packages/amplify_auth_cognito/example/lib/main.dart

```diff
@@ -346,6 +346,24 @@ class _MyAppState extends State<MyApp> {
     }
   }
 
+  void _getCurrentUser() async {
+    setState(() {
+      error = "";
+      exceptions = [];
+    });
+    try {
+      AuthUser res = await Amplify.Auth.getCurrrentUser();
+      print(res);
+    } on AuthError catch (e) {
```

Comment:
I'd encourage you to not use single letter variable names anywhere. They're very hard to search for with e.g. `less`, `vim`, even IDEs. How about `error`, or `authError`, for this example. But, same general advice around the code base.

Reply:
Agreed... but again this will require several changes across this file for consistency, so I'm going to do it in a separate PR.